tracing-fmt/src/default.rs

```diff
@@ -195,7 +205,12 @@ where
                 f.pad(":")?;
             }
             seen = true;
-            write!(f, "{}", Style::new().bold().paint(span.name()))
+
+            if self.1 {
```

Comment:
This is why I'd prefer to use named fields --- the meaning of `self.1` is extremely unclear at a glance.